tools/cockpit.spec

```diff
@@ -347,8 +347,8 @@ Cockpit support for reading PCP metrics and loading PCP archives.
 %package dashboard
 Summary: Cockpit SSH remoting and dashboard
 Requires: libssh >= %{libssh_version}
-Requires: cockpit-ws = %{version}-%{release}
-Provides: cockpit-ssh = %{version}-%{release}
+Requires: cockpit-bridge > 134
```

Comment:
I don't think we should artificially bump the requirements in manifest.json - only if the actual content demands it. Anything else is packaging magic/glue on top.